.prettierignore

```diff
@@ -0,0 +1,11 @@
+# Ignore artifacts:
+build
+coverage
+
+# Ignore these file types
+*.html
+*.md
+*.yaml
+*.css
+*.cjs
+*.lock
```

Comment:
These files don't need to be checked, let me know if I need to add or remove things from this list.

Reply:
I wonder instead of an exclude file, if we dont just want to have an include file list that includes js, jsx,ts,tsx ? I can't imagine any other file types we would want prettier to hit up? Perhaps just .rs if we are not using cargo fmt.

An include list will have fewer surprises if something gets added that we forgot to exclude?

Reply:
Do you have certain directories in mind, as this seems like a viable option: https://stackoverflow.com/a/70465911

Reply:
Yeah or we can just use the catchall 

**/*.{js,ts,jsx,tsx}

Perhaps with node_modules excluded in the exclude list? That should get everything that is a js, ts, jsx, tsx file not in node_modules.

or we could be more specific with 

**/src/**/*.{js,ts,jsx,tsx} which would format any files with those extensions in any sub folder that was named src.